case_studies/sdss_galaxies/config/config.yaml

```diff
@@ -36,13 +36,15 @@ datasets:
         batch_size: 64
         n_batches: 10
         background: 865
+        n_bands: 1
```

Comment:
Added a few defaults to configs (like this one) that weren't there before. If this isn't desirable, I can remove them and add those defaults back as they were. No change in behavior.

Reply:
I think this is a good change. We should err on the side of being explicit, especially since we have adding multiple bands on the road map.